docs/GraFx-Developers/grafx-studio/environment-api-quickstart/02-managing-integrations/index.md

```diff
@@ -0,0 +1,53 @@
+# Managing Integration
```

Comment:
Where is part 01?

Reply:
Sean told me, it wasn't ready yet. Will be in a future update.